test/common/test-functions.js

```diff
@@ -51,14 +51,6 @@ function ph_set_val (sel, val) {
     el.dispatchEvent(ev);
 }
 
-function ph_set_text_direction(sel, direction) {
```

Comment:
Ah, I remember that leftover. Thanks for cleaning up! Should have been a separate commit, but :man_shrugging: